doc/integrator/internationalization.rst

```diff
@@ -71,6 +70,24 @@ For non Docker project:
      * some metadata as disclaimer
      * editable layer (database structure, data or enumerations)
 
+.. note::
+
+   In mapfiles, attributes added by the mapserver's substitution will be
```

Comment:
In mapfiles, attributes added by mapserver substitution will not be collected for translation.